kalite/updates/views.py

```diff
@@ -2,7 +2,7 @@
 import os
 from annoying.decorators import render_to
 
-from django.utils.translation import ugettext_lazy
+from django.utils.translation import ugettext_lazy as _
```

Comment:
@rtibbles 
Thank you for the comments. I think you mean:
 from django.utils.translation import ugettext_lazy as _, ugettext_lazy